tests/setup/extension.rs

```diff
@@ -51,9 +52,11 @@ where
     /// This hook will add metadata into the request context.
     fn pre_request_hook(
```

Comment:
Exposed two new values in this hook: the name of the operation, and access to the database.

Reply:
To make sure I understand... what is the name of the operation?  Is that the name of the GraphQL top level Query or Mutation?

Reply:
You can give an operation an optional name. 

Anonymous query:

```
query {
  User {
    id
  }
}
```

Named query:

```
query MyQueryForUsers {
  User {
    id
  }
}
```

Reply:
Nice. Thank you. I have learned a new thing.